Makefile

```diff
@@ -41,14 +41,16 @@ temp-pc:
 	${SED} "s#/include/libgovarnam##g" govarnam.pc
 	${SED} "s#/lib\$$##g" govarnam.pc
 
-install-script:
+.PHONY: install-script
```

Comment:
I think we can drop this `.PHONY` then as it's unused.

Reply:
done!